CMakeSettings.json

```diff
@@ -0,0 +1,69 @@
+{
+    // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
+    "configurations": [
+        {
+        "name": "x86-Debug",
+        "generator": "Ninja",
```

Comment:
We should stop using makefiles (or ninja etc) and move to native `msbuild` tool to fine-tune build params (like building XP binaries)